plugins/docker/docker_

```diff
@@ -174,23 +372,42 @@ def main():
             print("graph_category virtualization")
             print("containers_quantity.label Containers")
         else:
-            print('containers_quantity.value', len(client.containers.list()))
+            print('containers_quantity.value', len(client.containers))
     elif wildcard == "images":
         if mode == "config":
             print("graph_title Docker images")
             print("graph_vlabel images")
             print("graph_category virtualization")
+            print("graph_total All images")
+            print("intermediate_quantity.label Intermediate images")
+            print("intermediate_quantity.draw AREASTACK")
+            print("intermediate_quantity.info All unused images can be deleted with "
+                  "`docker image prune --all`")
             print("images_quantity.label Images")
+            print("images_quantity.draw AREASTACK")
+            print("images_quantity.info Images can be used in containers with "
+                  "`docker container create --name <NAME> <IMAGE_ID >` or "
+                  "`docker container run --name <NAME> <IMAGE_ID> <COMMAND>`")
+            print("dangling_quantity.label Dangling images")
+            print("dangling_quantity.draw AREASTACK")
+            print("dangling_quantity.info Dangling images can be deleted with "
+                  "`docker image prune`"
+                  "or tagged with `docker image tag <IMAGE_ID> <NAME>`")
+            print("dangling_quantity.warning 10")
         else:
-            print('images_quantity.value', len(client.images.list()))
+            print_images_count(client)
     elif wildcard == "volumes":
         if mode == "config":
             print("graph_title Docker volumes")
             print("graph_vlabel volumes")
             print("graph_category virtualization")
             print("volumes_quantity.label Volumes")
+            print("volumes_quantity.draw AREASTACK")
+            print("volumes_quantity.info Unused volumes can be deleted with "
```

Comment:
Here you are using *two* spaces (instead of one) due to your line breaking.

But this is very minor and I would have omitted to comment on it, if there would not be a [style-check failure](https://travis-ci.org/github/munin-monitoring/contrib/builds/720628638) :)

Reply:
Hum, I don't see the added space here. 

However, there were a few other test failures that were valid, so I fixed those (: